packages-ui/roosterjs-react/lib/ribbon/plugin/createRibbonPlugin.ts

```diff
@@ -49,6 +50,21 @@ class RibbonPluginImpl implements RibbonPlugin {
      * @param event PluginEvent object
      */
     onPluginEvent(event: PluginEvent) {
+        if (
+            event.eventType == PluginEventType.KeyDown &&
+            event.rawEvent.key == 'k' &&
+            (event.rawEvent.ctrlKey || event.rawEvent.metaKey) &&
```

Comment:
you can use function `isCtrlOrMetaPressed()` (packages\roosterjs-editor-dom\lib\event\isCtrlOrMetaPressed.ts) which will also help check if current OS is Mac. Because on Window metakey can be true when Win key is pressed.